exercises/concept/train-driver/.docs/instructions.md

```diff
@@ -0,0 +1,111 @@
+# Instructions
+
+Your friend is a train driver and has to drive cargo trains between cities.
+Although your friend isn't amazing with handling the computers and would like some help with it.
+Your friend would like your help organizing the train and correcting the mistakes in the data.
+
+
+```exercism/note
+To practice, use a for the rest or spread to solve each of the tasks below.
+```
+
+## 1. Convert the data to an array
+
+Your friend has been keeping track of how much each wagon weighs. Although they are not sure how many wagons and would like the data to be returned as an array.
+
+```exercism/note
+
+Implement a function `getListOfWagons` that accepts an unknown amount of whole numbers that contains the weight of each wagon.
```

Comment:
whole -> integer

Reply:
I am a bit unsure since integer atleast for me implies that the number could be negative but if we go for the story there the locomotive has index 1 shouln't it be more accurate to say positive whole number?

Reply:
then it should be "positive integer", my main point is that we try to use the correct terms usually used in programming and that would be integer here

Reply:
True